src/modules/reading-room/components/AddToCollectionBlade/AddToCollectionBlade.tsx

```diff
@@ -113,7 +116,7 @@ const AddToCollectionBlade: FC<AddToCollectionBladeProps> = (props) => {
 				};
 				if (pair.checked) {
 					return collectionsService
-						.addToCollection(pair.id, selected.schemaIdentifier)
+						.addToCollection(pair.collection, selected.schemaIdentifier)
```

Comment:
follow the glossay when you rename stuff:
We're following the english names in the "DB TO BE" column.

so this would be: "folder" or maybe "folderId"